WordPress/Classes/ViewRelated/NUX/SignupViewController.swift

```diff
@@ -469,10 +470,13 @@ import WordPressShared
 
 
     @IBAction func handleTermsOfServiceButtonTapped(_ sender: UIButton) {
-        let url = URL(string: WPAutomatticTermsOfServiceURL)!
-        let controller = WebViewControllerFactory.controller(url: url)
-        let navController = RotationAwareNavigationViewController(rootViewController: controller)
-        present(navController, animated: true, completion: nil)
+        guard let url = URL(string: WPAutomatticTermsOfServiceURL) else {
+            fatalError()
```

Comment:
Do we really want a fatal error here? Should the app crash?

Reply:
Although this is *not* a Critical Error, reasoning behind that fatalError clause there is:

- The URL instantiation should *never* fail
- If it ever does, we'd get a Crashlytics log, and we'd get to fix it

Still, i (now!) feel like it may have been an overkill. I'll switch that. Thanks for bringing this up James!